pages/_document.js

```diff
@@ -0,0 +1,19 @@
+import { Html, Head, Main, NextScript } from "next/document";
+
+export default function Document() {
+    return (
+        <Html lang="en">
+            <Head />
+            <link
+                href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css"
+                rel="stylesheet"
+                integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65"
+                crossorigin="anonymous"
+            />
```

Comment:
This could have been on the same file where you insert scripts assuming using CDN is the best way to import boostrap.

Reply:
It doesn’t work that way when using bootstrap CDN